ts/test/test-time-profiler.ts

```diff
@@ -28,20 +28,6 @@ const PROFILE_OPTIONS = {
 };
 
 describe('Time Profiler', () => {
-  describe('profile', () => {
-    it('should detect program or idle time', async () => {
```

Comment:
Should the opposite test be added?

Reply:
So the reason I removed it rather than altered it is because previously it would have been the case that at least one of the two of `(idle)` or `(program)` would be present. With `(idle)` removed `(program)` may appear inconsistently and I don't think there's a good way to make it consistently show `(program)` to keep the test.

Reply:
Could at least a test ensuring `(idle)` is not present be kept? For `(program)` it can always be addressed later if needed.

Reply:
No way to be sure `(idle)` is present either though. Especially given that not all versions would even _have_ `(idle)` in the first place.

Reply:
We want to be sure that it's _not_ present, so it should be fine.

Reply:
My point though is there's no control case to test against though. There's no scenario in which _broken_ functionality would be certain to get `(idle)` layers.

Reply:
It would be testing that there is no regression. Since there was previously a test checking for its presence, it means that a regression could happen that brings it back. Adding the opposite test thus makes sense. It's also worth noting that it's basically the same test, but with a different description and a slightly different implementation so it's not a change that would take a lot of time to do and need justification for the ROI.